### PR TITLE
Add cookie settings button in CookieListBlock

### PR DESCRIPTION
**Pull request by gustaveen**
## Describe your changes
Add cookie settings button in CookieListBlock

https://github.com/HedvigInsurance/racoon/assets/6661511/0359bf20-5369-4da3-a8d4-b3413e34f201

## Justify why they are needed
Compliance

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by gustaveen** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3166** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3166" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3166?utm_source=stack-comment).

File: apps/store/src/blocks/CookieListBlock.tsx [hunk 1]
```diff
@@ -18,6 +21,32 @@ export const CookieListBlock = () => {
  * to migrate between OneTrust and the component
  */
 
+const PreferenceButton = styled.button({
+  '&#ot-sdk-btn.ot-sdk-show-settings': {
```
Comment (gustaveen) [inline apps/store/src/blocks/CookieListBlock.tsx:25]: Needed to override styles from OneTrust

File: apps/store/src/blocks/CookieListBlock.tsx [hunk 2]
```diff
@@ -6,6 +6,9 @@ export const CookieListBlock = () => {
   return (
     <GridLayout.Root>
       <GridLayout.Content width={{ base: '1', md: '2/3', xxl: '1/2' }} align={'center'}>
+        <PreferenceButton id="ot-sdk-btn" className="ot-sdk-show-settings">
+          Cookies Settings
```
Comment (gustaveen) [inline apps/store/src/blocks/CookieListBlock.tsx:10]: Suggested to have a fallback text, translated text is handled by OneTrust script

Comment (robinandeer) [reply apps/store/src/blocks/CookieListBlock.tsx:10]: nit: we could add a code comment about this

**Review by robinandeer — APPROVED** [review]
Looks good!

Added via Giphy

**Pull request merged**